rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -316,24 +316,33 @@ object Reduce {
       for {
         quote   <- eval(send.chan)
         subChan <- substituteAndCharge[Quote, M](quote, 0, env, costAccountingAlg)
-        unbundled <- subChan.value.singleBundle() match {
-                      case Some(value) =>
-                        if (!value.writeFlag) {
-                          s.raiseError(ReduceError("Trying to send on non-writeable channel."))
-                        } else {
-                          s.pure(Quote(value.body))
-                        }
-                      case None => Applicative[M].pure(subChan)
-                    }
-
-        data <- send.data.toList.traverse(x => evalExpr(x))
-        substData <- data.traverse(
-                      p =>
-                        substituteAndCharge[Par, M](p, 0, env, costAccountingAlg)
-                          .map(par => Channel(Quote(par)))
-                    )
-        _ <- produce(unbundled, substData, send.persistent, rand)
-        _ <- costAccountingAlg.charge(SEND_EVAL_COST)
+        _ <- {
+          if (ParIsForgeable.isForgeable(subChan.value))
+            Applicative[M].pure(())
```

Comment:
Please add a test showing that sending on a forgeable channel is a NOOP. So far we only have tests that check various cases when sending to a channel isn't a noop.

Reply:
Done.